lte/gateway/c/core/BUILD.bazel

```diff
@@ -0,0 +1,1163 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+load("//bazel:asn1c.bzl", "cc_asn1_library")
+
+package(default_visibility = ["//lte/gateway/c/core:__subpackages__"])
+
+cc_asn1_library(
+    name = "asn1_r15",
+    asn1_file = "oai/tasks/s1ap/messages/asn1/r15/s1ap-15.6.0.asn1",
+    prefix = "S1ap_",
+)
+
+cc_asn1_library(
+    name = "asn1_r16",
+    asn1_file = "oai/tasks/ngap/messages/asn1/r16/r16.asn1",
+    prefix = "Ngap_",
+)
+
+// TODO(#11892): Giant Bazel label is bad practice, break this up
```

Comment:
needs # for comment - breaks the bazel builds in CI

Reply:
Whoops! Added that late last night and didn't re-test LOL. Thanks.